pkg/nvcdi/driver-nvml.go

```diff
@@ -100,7 +106,11 @@ func NewDriverLibraryDiscoverer(logger logger.Interface, driver *root.Driver, nv
 	hooks, _ := discover.NewLDCacheUpdateHook(logger, libraries, nvidiaCDIHookPath, ldconfigPath)
 
 	d := discover.Merge(
-		libraries,
```

Comment:
Did you intend to remove `libraries` from here? My understanding is that this PR is adding another hook (to create the `libcuda.so` symlink), not removing any of the discoverers in this list.